2021/Robot2021/src/main/cpp/commands/LEDSet.cpp

```diff
@@ -12,6 +12,8 @@
 
 #include "commands/LEDSet.h"
 
+#include "spdlog/spdlog.h"

```

Comment:
```suggestion
#include <spdlog/spdlog.h>
```

Reply:
Also, the #include line is within the RobotBuilder "guards" // BEGIN ... // END. That means this inlude will be deleted the next time RobotBuilder gets run.